apps/wallet/tests/utils/localnet.ts

```diff
@@ -3,12 +3,30 @@
 
 // eslint-disable-next-line import/order
 import 'tsconfig-paths/register';
-import { Ed25519Keypair } from '@mysten/sui.js';
+import {
+    Ed25519Keypair,
+    JsonRpcProvider,
+    type Keypair,
+    localnetConnection,
+} from '@mysten/sui.js';
 import * as bip39 from '@scure/bip39';
 import { wordlist } from '@scure/bip39/wordlists/english';
 
+export async function generateKeypairFromMnemonic(mnemonic: string) {
+    return Ed25519Keypair.deriveKeypair(mnemonic);
+}
+
 export async function generateKeypair() {
     const mnemonic = bip39.generateMnemonic(wordlist);
-    const keypair = Ed25519Keypair.deriveKeypair(mnemonic);
+    const keypair = await generateKeypairFromMnemonic(mnemonic);
     return { mnemonic, keypair };
 }
+
+export function generateAddress(keypair: Keypair) {
+    return keypair.getPublicKey().toSuiAddress();
+}
+
+export async function requestingSuiFromFaucet(address: string) {
```

Comment:
This is only used once, not convinced this needs to be it's own helper

Reply:
Chat offline, this is not working if called directly in test files, might be other issues but resolved for now.